### PR TITLE
Update download.sh

### PR DESCRIPTION
**Pull request by hyungupark**
modify for CPU_ARCH not found

### CODE REVIEW

**Comment by GDemay** [comment]
Could you link to the issue? Thanks

**Comment by hyungupark** [comment]
this commit fixes #1140

**Review by samuelselvan — APPROVED** [review]

**Review by Kaktrollet — COMMENTED** [review]
C

File: download.sh [hunk 1]
```diff
@@ -62,9 +62,10 @@ do
     wget --continue ${PRESIGNED_URL/'*'/"${MODEL_PATH}/params.json"} -O ${TARGET_FOLDER}"/${MODEL_PATH}/params.json"
     wget --continue ${PRESIGNED_URL/'*'/"${MODEL_PATH}/checklist.chk"} -O ${TARGET_FOLDER}"/${MODEL_PATH}/checklist.chk"
     echo "Checking checksums"
-    if [ "$CPU_ARCH" = "arm64" ]; then
+    CPU_ARCH=$(uname -m)
```
Comment (ruidazeng) [inline download.sh:65]: Why is this necessary? `CPU_ARCH=$(uname -m)` was set on line 25 once already.

Comment (hyungupark) [reply download.sh:65]: Oh, I didn't see it, thank you!
